web/blog/matching-yara-rules-in-byte-pipelines/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Matching Yara Rules in Byte Pipelines
+authors:
+   - mavam
+date: 2023-10-24
+tags: [yara, operator, dfir, detection engineering]
+comments: true
+---
+
+The new [`yara`][yara-operator] operator matches [YARA][yara] rules on byte
+pipelines, producing a structured match output to conveniently integrate
```

Comment:
I think the term "byte pipeline" is a bit confusing. The entire thing is the pipeline; the `yara` operator is matching Yara rules on bytes, but not on pipelines.

Reply:
Can you make a suggestion? I'm not seeing the confusion here and don't know in what direction to correct.

Reply:
I think just "on bytes" is better than "on byte pipelines."

Reply:
Thanks, I used that instead.